src/components/navigation/presentational/navItems.js

```diff
@@ -34,8 +35,8 @@ const NavItems = props => {
       <Column flex={3} alignItems="center" justifyContent="center">
         <img src={logo} alt="" />
       </Column>
-      {listItems.map((items, index) => (
-        <Column key={index}>
+      {listItems.map(items => (
+        <Column key={shortid.generate()}>
```

Comment:
nit: these again will be remounted on every render. Not sure why the index wasn't just fine? Nav items are static anyway, right?

Reply:
as discussed we'll leave this for now as using index is antipattern